contentcuration/contentcuration/management/commands/exportchannel.py

```diff
@@ -369,6 +372,12 @@ def write_assessment_item(assessment_item, zf):
     result = render_to_string(template, context).encode('utf-8', "ignore")
     write_to_zipfile("{0}.json".format(assessment_item.assessment_id), result, zf)
 
+def process_formulas(content):
+    for match in re.finditer(ur'\$(\$[^\$]+\$)\$', content):
```

Comment:
Do we disallow literal `$`s (`\\$`) inside the formula text? If we don't, this regex will foul up. (I am hoping we don't, and don't think we should!)